### ISSUE TITLE
Starting with a space in Exclude Files on the jslint4java preference page will cause all files to be excluded

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#19]
```
Workaround: No spaces in before starting list of patterns

```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 24 Feb 2011 at 10:34

**Comment by GoogleCodeExporter** [comment]
```
Needed trim in JSLintBuilder.java
```

Original comment by `paul.beu...@gmail.com` on 5 Feb 2012 at 8:13
- Changed state: **Started**
- Added labels: **Milestone-Release1.2.7**

**Comment by GoogleCodeExporter** [comment]
```
Released with 1.2.7 February 7, 2012
```

Original comment by `paul.beu...@gmail.com` on 7 Feb 2012 at 4:10
- Changed state: **Fixed**
